src/main/java/org/prebid/server/spring/config/PrometheusMapperConfiguration.java

```diff
@@ -0,0 +1,297 @@
+        mapperConfigs.addAll(privacyMappingConfigs());
+        mapperConfigs.addAll(vertxResponseConfig);
+        mapperConfigs.addAll(vertxRequestConfigs);
+        mapperConfigs.add(vertxServer);
+
+        return new CustomMappingSampleBuilder(mapperConfigs);
+    }
+
+    // example: requests.ok.openrtb2-web
+    private final MapperConfig requestsOk = new MapperConfig(
+            "requests.*.*",
+            "requests.status.type",
+            Map.of("status", "${0}", "type", "${1}")
+    );
+
+    // example: bidder-cardinality.4.requests
+    private final MapperConfig bidderCardinality = new MapperConfig(
+            "bidder-cardinality.*.requests",
+            "bidder-cardinality.requests",
+            Map.of("cardinality", "${0}"));
+
+    // example: vertx.http.clients.get-requests
+    // example: vertx.http.clients.post-requests
+    private final List<MapperConfig> vertxRequestConfigs = Stream.of(
+            "get", "post", "head", "delete", "connect", "options", "patch", "put", "other", "trace"
+    ).map(request -> new MapperConfig(
+                    "vertx.http.clients." + request + "-requests",
+                    "vertx.http.clients.requests_by_type",
+                    Map.of("request", request)
+            )
+
+    ).collect(Collectors.toList());
+
+    private final List<MapperConfig> vertxResponseConfig = Stream.of(
+            "1xx", "2xx", "3xx", "4xx"
+    ).map(response -> new MapperConfig(
+                    "vertx.http.clients.responses-" + response,
+                    "vertx.http.clients.response_by_status",
+                    Map.of("responses", response)
+            )
+
+    ).collect(Collectors.toList());
+
+    // example: vertx.http.servers.[IP]:[PORT].delete-requests
+    // Another option would be to match over the different ports and use a reasonable tag name
+    // However ":" is not supported in the matching reqex of the mapper config
+    private final MapperConfig vertxServer = new MapperConfig(
+            "vertx.http.servers.*.*.*.*.*", "vertx.http.servers.${4}",
+            Map.of("address", "${0}.${1}.${2}.${3}"));
+
+    private List<MapperConfig> privacyMappingConfigs() {
+        final MapperConfig privacyTcfMissing = new MapperConfig(
+                "privacy.tcf.missing",
+                "privacy.tcf.errors",
+                Map.of("error", "missing"));
+
+        final MapperConfig privacyTcfInvalid = new MapperConfig(
+                "privacy.tcf.invalid",
+                "privacy.tcf.errors",
+                Map.of("error", "invalid"));
+
+        final MapperConfig privacyVendorList = new MapperConfig(
+                "privacy.tcf.*.vendorlist.*",
+                "privacy.tcf.vendorlist",
+                Map.of("tcf", "${0}", "status", "${1}"));
+
+        final MapperConfig privacyTcfStatus = new MapperConfig(
+                "privacy.tcf.*.*",
+                "privacy.tcf.${1}",
+                Map.of("tcf", "${0}"));
```

Comment:
All of these should be separate beans. Could be extracted to separate configuration.